docs/errors/_err_list_manual.md

```diff
@@ -50,21 +50,25 @@ https://dashboard.ngrok.com/get-started/your-authtoken</td>
 			<td>
 				<a id="ERR_NGROK_107" href="/docs/errors/err_ngrok_107">ERR_NGROK_107</a>
 			</td>
-			<td class="pre-wrapped">The authtoken you specified is properly formed, but it is invalid.
-Your authtoken: <code>&lt;TOKEN&gt;</code>
-This usually happens when:
-    - You reset your authtoken
-    - Your authtoken was for a team account that you were removed from
-    - You are using ngrok link and this credential was explicitly revoked
-Go to your ngrok dashboard and double check that your authtoken is correct:
-https://dashboard.ngrok.com/get-started/your-authtoken</td>
+			<td class="pre-wrapped">
```

Comment:
MDX 3 cares about the whitespace here. This is mostly formatting, no actual escaping